pkg/subctl/cmd/join.go

```diff
@@ -71,6 +72,7 @@ func addJoinFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&ipsecDebug, "ipsec-debug", false, "Enable IPsec debugging (verbose logging)")
 	cmd.Flags().BoolVar(&submarinerDebug, "subm-debug", false, "Enable Submariner debugging (verbose logging)")
 	cmd.Flags().BoolVar(&noLabel, "no-label", false, "skip gateway labeling")
+	cmd.Flags().StringVar(&kubefedController, "kubefed-controller", "", "overrides the use of host-cluster-context name in resource names created in the target cluster.")
```

Comment:
The name "kubefed-controller" seems out of context with the description - how about we name it "kubefed-host-cluster-context"? Similarly the variable "kubefedHostClusterContext".

Reply:
Done.